cypress/fixtures/HeaderBar/getModulesWithSpecialChars.json

```diff
@@ -0,0 +1,229 @@
+            "displayName": "A \\s character",
+            "icon": "../icons/dhis-web-data-visualizer.png",
+            "description": ""
+        },
+        {
+            "name": "\\",
+            "namespace": "/\\",
+            "defaultAction": "../\\.action",
+            "displayName": "A \\ character",
+            "icon": "../icons/dhis-web-data-visualizer.png",
+            "description": ""
+        },
+
+        {
+            "name": "dhis-web-dashboard",
+            "namespace": "/dhis-web-dashboard",
+            "defaultAction": "../dhis-web-dashboard/index.action",
+            "displayName": "Dashboard",
+            "icon": "../icons/dhis-web-dashboard.png",
+            "description": ""
+        },
+        {
+            "name": "dhis-web-capture",
+            "namespace": "/dhis-web-capture",
+            "defaultAction": "../dhis-web-capture/index.action",
+            "displayName": "Capture",
+            "icon": "../icons/dhis-web-capture.png",
+            "description": ""
+        },
+        {
+            "name": "dhis-web-maintenance",
+            "namespace": "/dhis-web-maintenance",
+            "defaultAction": "../dhis-web-maintenance/index.action",
+            "displayName": "Maintenance",
+            "icon": "../icons/dhis-web-maintenance.png",
+            "description": ""
+        },
+        {
+            "name": "dhis-web-maps",
+            "namespace": "/dhis-web-maps",
+            "defaultAction": "../dhis-web-maps/index.action",
+            "displayName": "Maps",
+            "icon": "../icons/dhis-web-maps.png",
+            "description": ""
+        },
+        {
+            "name": "dhis-web-event-reports",
+            "namespace": "/dhis-web-event-reports",
+            "defaultAction": "../dhis-web-event-reports/index.action",
+            "displayName": "Event Reports",
+            "icon": "../icons/dhis-web-event-reports.png",
+            "description": ""
+        },
+        {
+            "name": "dhis-web-interpretation",
+            "namespace": "/dhis-web-interpretation",
+            "defaultAction": "../dhis-web-interpretation/index.action",
+            "displayName": "Interpretations",
+            "icon": "../icons/dhis-web-interpretation.png",
+            "description": ""
+        },
+        {
+            "name": "dhis-web-importexport",
+            "namespace": "/dhis-web-importexport",
+            "defaultAction": "../dhis-web-importexport/index.action",
+            "displayName": "Import/Export",
+            "icon": "../icons/dhis-web-importexport.png",
+            "description": ""
+        },
+        {
+            "name": "WHO Metadata browser",
+            "namespace": "WHO Metadata browser",
+            "defaultAction": "https://play.dhis2.org/2.32.0/api/apps/WHO-Metadata-browser/index.html",
+            "displayName": "",
+            "icon": "https://play.dhis2.org/2.32.0/api/apps/WHO-Metadata-browser/icons/medicine-48.png",
```

Comment:
These should be `https://debug.dhis2.org/dev` since `2.32.0` instance isn't permanent (no longer exists on play) and play prevents cross-domain cookies